src/Softeq.NetKit.Chat.Data.Persistent.Sql/Repositories/ClientRepository.cs

```diff
@@ -68,10 +91,33 @@ public async Task<Client> GetClientWithMemberAsync(string clientConnectionId)
         {
             using (var connection = _sqlConnectionFactory.CreateConnection())
             {
-                var sqlQuery = @"SELECT *
-                                 FROM Clients c 
-                                 INNER JOIN Members m ON c.MemberId = m.Id
-                                 WHERE c.ClientConnectionId = @clientConnectionId";
+                var sqlQuery = $@"
+                    SELECT 
+                        c.{nameof(Client.Id)},
+                        c.{nameof(Client.ClientConnectionId)},
+                        c.{nameof(Client.LastActivity)},
+                        c.{nameof(Client.LastClientActivity)},
+                        c.{nameof(Client.Name)},
+                        c.{nameof(Client.UserAgent)},
+                        c.{nameof(Client.MemberId)},
+                        m.{nameof(Member.Id)},
+                        m.{nameof(Member.Email)},
+                        m.{nameof(Member.IsAfk)},
+                        m.{nameof(Member.IsBanned)},
+                        m.{nameof(Member.LastActivity)},
+                        m.{nameof(Member.LastNudged)},
+                        m.{nameof(Member.Name)},
+                        m.{nameof(Member.PhotoName)},
+                        m.{nameof(Member.Role)},
+                        m.{nameof(Member.SaasUserId)},
+                        m.{nameof(Member.Status)},
+                        m.{nameof(Member.IsActive)}
+                    FROM 
```

Comment:
When we'll merge pull request with new chat version, we need to be sure that we'll add new property:
m.{nameof(Member.IsDeleted)}